bitnami/redis/templates/metrics-sentinel-svc.yaml

```diff
@@ -0,0 +1,35 @@
+{{- if and .Values.sentinel.enabled .Values.sentinel.metrics.enabled }}
+apiVersion: v1
+kind: Service
+metadata:
+  name: {{ template "redis.fullname" . }}-sentinel-metrics
+  namespace: {{ .Release.Namespace | quote }}
+  labels:
+    app: {{ template "redis.name" . }}
+    chart: {{ template "redis.chart" . }}
+    release: {{ .Release.Name }}
+    heritage: {{ .Release.Service }}
+    app.kubernetes.io/component: "sentinel-metrics"
+    {{- if .Values.sentinel.metrics.service.labels -}}
+    {{- toYaml .Values.sentinel.metrics.service.labels | nindent 4 }}
+    {{- end -}}
+  {{- if .Values.sentinel.metrics.service.annotations }}
+  annotations: {{- toYaml .Values.sentinel.metrics.service.annotations | nindent 4 }}
+  {{- end }}
+spec:
+  type: {{ .Values.sentinel.metrics.service.type }}
+  {{- if eq .Values.sentinel.metrics.service.type "LoadBalancer" }}
+  externalTrafficPolicy: {{ .Values.sentinel.metrics.service.externalTrafficPolicy }}
+  {{- end }}
+  {{- if and (eq .Values.sentinel.metrics.service.type "LoadBalancer") .Values.sentinel.metrics.service.loadBalancerIP }}
+  loadBalancerIP: {{ .Values.sentinel.metrics.service.loadBalancerIP }}
+  {{- end }}
+  ports:
+    - port: {{ .Values.sentinel.metrics.service.port }}
+      targetPort: sentinelmetrics
+      protocol: TCP
+      name: sentinelmetrics
```

Comment:
Should we also adjust the port naming for Istio as it was already started in #5679. So that the metrics port for sentinel is prefixed with `http` to avoid the warnings from `istioctl analyze`?

```sh
Info [IST0118] (Service redis-metrics.backend) Port name metrics (port: 9121, targetPort: metrics) doesn't follow the naming convention of Istio port.
Info [IST0118] (Service redis-sentinel-metrics.backend) Port name sentinelmetrics (port: 9355, targetPort: sentinelmetrics) doesn't follow the naming convention of Istio port.
```

@migruiz4 I saw you already did some adjustments, should I change this or do you want to do it?